<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAreaCamPage</class>
 <widget class="QWidget" name="CAreaCamPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="maximumSize">
        <size>
         <width>240</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>图象设置</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="6" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>单相机分辨率Y</string>
          </property>
         </widget>
        </item>
        <item row="23" column="1">
         <widget class="QPushButton" name="pushButton_continue">
          <property name="text">
           <string>连续采集</string>
          </property>
         </widget>
        </item>
        <item row="10" column="2">
         <widget class="QSpinBox" name="spinBox_ImageWidth">
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>9999999</number>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QSpinBox" name="spinBox_resolutionY">
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>9999999</number>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>单相机分辨率X</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0" colspan="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>图像总Height(pix)</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QComboBox" name="comboBox_cam_name">
          <item>
           <property name="text">
            <string>Cam1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Cam2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Cam3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Cam4</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="10" column="0" colspan="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>图像总Width(pix)</string>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="QSpinBox" name="spinBox_overlapX">
          <property name="maximum">
           <number>99999</number>
          </property>
         </widget>
        </item>
        <item row="16" column="0" colspan="2">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>相机触发模式</string>
          </property>
         </widget>
        </item>
        <item row="23" column="0">
         <widget class="QPushButton" name="pushButton_snap">
          <property name="text">
           <string>单张采集</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0" colspan="2">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>相机串号(SN)</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>X方向重叠pix</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QCheckBox" name="checkBox_enable_grap">
          <property name="text">
           <string> 使能相机采图</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="13" column="0" colspan="2">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>相机曝光(us)</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0" colspan="2">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Y方向重叠pix</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QSpinBox" name="spinBox_resolutionX">
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string/>
          </property>
          <property name="maximum">
           <number>99999999</number>
          </property>
          <property name="value">
           <number>8196</number>
          </property>
         </widget>
        </item>
        <item row="12" column="2">
         <widget class="QLineEdit" name="lineEdit_cam_sn">
          <property name="text">
           <string>Cam1</string>
          </property>
         </widget>
        </item>
        <item row="11" column="2">
         <widget class="QSpinBox" name="spinBox_ImageHeight">
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>9999999</number>
          </property>
         </widget>
        </item>
        <item row="16" column="2">
         <widget class="QComboBox" name="comboBox_trig_mode">
          <item>
           <property name="text">
            <string>关闭</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>外触发</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>软触发</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="13" column="2">
         <widget class="QSpinBox" name="spinBox_cam_expose">
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>9999999</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QCheckBox" name="checkBox_enable_save_raw">
          <property name="text">
           <string> 使能原图保存</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QSpinBox" name="spinBox_overlapY">
          <property name="maximum">
           <number>999999</number>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>相机个数</string>
          </property>
         </widget>
        </item>
        <item row="14" column="0" colspan="2">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>相机增益</string>
          </property>
         </widget>
        </item>
        <item row="17" column="0" colspan="2">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>触发电平</string>
          </property>
         </widget>
        </item>
        <item row="15" column="2">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_delay_trig">
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <double>99999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
        <item row="15" column="0" colspan="2">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>延时触发距离(mm)</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QSpinBox" name="spinBox_camCnt">
          <property name="maximum">
           <number>4</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="17" column="2">
         <widget class="QComboBox" name="comboBox_trig_edge">
          <item>
           <property name="text">
            <string>上升沿</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>下降沿</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="23" column="2">
         <widget class="QPushButton" name="pushButton_stop">
          <property name="text">
           <string>停止采集</string>
          </property>
         </widget>
        </item>
        <item row="20" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>触发IO名</string>
          </property>
         </widget>
        </item>
        <item row="19" column="2">
         <widget class="QSpinBox" name="spinBox_channel_cnt">
          <property name="maximum">
           <number>8</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>相机名称</string>
          </property>
         </widget>
        </item>
        <item row="19" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>通道数</string>
          </property>
         </widget>
        </item>
        <item row="14" column="2">
         <widget class="QSpinBox" name="spinBox_cam_gain">
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>9999999</number>
          </property>
         </widget>
        </item>
        <item row="22" column="2">
         <widget class="QSpinBox" name="spinBox_AcqTimeOut">
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string/>
          </property>
          <property name="maximum">
           <number>999999</number>
          </property>
          <property name="value">
           <number>6500</number>
          </property>
         </widget>
        </item>
        <item row="22" column="0" colspan="2">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>取图超时时间(ms)</string>
          </property>
         </widget>
        </item>
        <item row="20" column="2">
         <widget class="QLineEdit" name="lineEdit_triggle_IO">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="18" column="2">
         <widget class="QComboBox" name="comboBox_trig_index">
          <item>
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="18" column="0" colspan="2">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>触发控制器编号</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ShowPicture" name="widget_show_image" native="true">
     <property name="minimumSize">
      <size>
       <width>480</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>68</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ShowPicture</class>
   <extends>QWidget</extends>
   <header location="global">uitoolbox/showpicturewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
